sql/mob_skill_lists.sql

```diff
@@ -125,13 +125,8 @@ INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ifrit',38,844);
 INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ifrit',38,845);
 INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ifrit',38,846);
 INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ifrit',38,847);
--- INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ixion',39,1542);
-INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ixion',39,2334);
-INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ixion',39,2335);
-INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ixion',39,2336);
-INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ixion',39,2337);
-INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ixion',39,2338);
-INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ixion',39,2369);
+INSERT INTO `mob_skill_lists` VALUES ('Avatar-Ixion',39,2341); -- di hoof attack
```

Comment:
looks like these 2 IDs are special melee, which you are correct to separate from regular tp-skills but should be named in a way that identifies the skill list is expressly for that purpose (see other special melee skill lists such as trust Shantotto II or August, or Wyrms in Flight mode).

So you'd wind up with 2 lists named like (just example) : 
1. "Dark_Ixion"
2. "Dark_Ixion_Melee"

Reply:
Fair enough. I actually don't use a separate skill list. The logic for which skill to use is done in the mob lua